script.js

```diff
@@ -0,0 +1,81 @@
+    Speakerprofession: 'Global technology leader Jacky Wright has made a broad impact in business transformation and on social issues during two decades of technology leadership for companies including GE, BP and Microsoft, as well as the UK Government.',
+    Speaker1img: './images/speaker2.webp',
+    SpeakerDesign: './images/backgroundpattern4.jpg',
+  },
+  {
+    id: '3',
+    speakername: 'Susan Wojcicki',
+    Speakerprofile: 'CEO YouTube',
+    Speakerprofession: 'Susan Wojcicki has been fully immersed in the world of technology since being hired in 1999 as Google employee number 16 and has worked on everything from AdSense and Google Analytics to Google Books and Google Images, serving as SVP of AdWords and AdSense for 14 years.',
+    Speaker1img: './images/speaker3.webp',
+    SpeakerDesign: './images/backgroundpattern4.jpg',
+  },
+  {
+    id: '4',
+    speakername: 'Sheryl Sandberg',
+    Speakerprofile: 'COO Meta',
+    Speakerprofession: 'Arguably one of the most famous female faces in the world of tech, Sheryl Sandberg has held the position of Chief Operating Officer at Meta (formerly Facebook) for the last 14 years, where she has helped to dramatically increase the social media firm’s revenue.',
+    Speaker1img: './images/speaker44.jpg',
+    SpeakerDesign: './images/backgroundpattern4.jpg',
+  },
+  {
+    id: '5',
+    speakername: 'Jo-ann de Pass Olsovsky',
+    Speakerprofile: 'EVP & CIO Salesforce',
+    Speakerprofession: 'As EVP and chief information officer at Salesforce since 2018, Jo-ann de Pass Olsovsky leads the global IT function for Salesforce core, Mulesoft, Tableau and Slack, including technology strategy, architecture, applications, engineering, M&A systems integrations, data/analytics, network infrastructure',
+    Speaker1img: './images/speaker55.jpg',
+    SpeakerDesign: './images/backgroundpattern4.jpg',
+  },
+  {
+    id: '6',
+    speakername: 'Gwynne Shotwell',
+    Speakerprofile: 'President and COO SpaceX',
+    Speakerprofession: 'As President and COO of SpaceX, Gwynne Shotwell is responsible for the day-to-day operations of the space exploration company, as well as managing customer and strategic relations to support company growth.',
+    Speaker1img: './images/speaker66.jpg',
+    SpeakerDesign: './images/backgroundpattern4.jpg',
+  },
+];
+
+const featuredContainer = document.querySelector('.featured');
+
+FeaturedSpeaker.forEach((pro) => {
+  const speaker = `
+    <div class="speaker1 d-flex">
+    <div class="speaker1-pic">
+      <div class="design"> <img src=${pro.SpeakerDesign} alt=""></div>
+      <div class="img-speaker"><img src=${pro.Speaker1img} alt="" class="image"></div>
+    </div>
+    <div class="speaker1-items">
+      <div class="speaker-nam h5">${pro.speakername}</div>
+      <div class="speaker-profe">${pro.Speakerprofile}</div>
+      <div class="about-speaker">${pro.Speakerprofession}</div>
+    </div>
+  </div>
+    `;
+  featuredContainer.innerHTML += speaker;
```

Comment:
- [ ] Please use the proper 2-column grid layout here in this section. It should remain a 2-column grid layout on desktop screens. Currently, This is how it looks on my PC 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/198260498-b4fdc6ca-0a08-469f-9855-98409b087939.png)
**Given design**
![image](https://user-images.githubusercontent.com/90222110/198260566-b53ee91e-1908-4553-8e3d-66c86e1aa618.png)

As you can see, the layout is different. Please correct it 😃 👍 

- [ ] In addition, please reduce the size of each card here in this section so that they can fit well on the screen. You can see the look on the given design 😎